freeipa/src/test/java/com/sequenceiq/freeipa/service/stack/RepairInstancesServiceTest.java

```diff
@@ -473,6 +473,7 @@ public void testRebuildThrowsWhenSourceStackIsRunning() throws Exception {
 
         when(stackService.getByCrnAndAccountIdEvenIfTerminated(ENVIRONMENT_ID1, ACCOUNT_ID, FREEIPA_CRN)).thenReturn(stack);
         when(stackService.findByEnvironmentCrnAndAccountId(ENVIRONMENT_ID1, ACCOUNT_ID)).thenReturn(Optional.of(stack));
+        when(entitlementService.isFreeIpaRebuildEnabled(eq(ACCOUNT_ID))).thenReturn(true);
```

Comment:
Would it make sense to have a test when this is disabled, to check it throws the expected exception?

Reply:
Ooh good catch, I second this for completeness.

Reply:
ok, i added the tests;